src/native/hmac.rs

```diff
@@ -8,9 +8,14 @@ use crate::hash;
 use crate::hmac::*;
 use crate::interface::*;
 use crate::mechanism::*;
+use crate::misc::zeromem;
 
 use constant_time_eq::constant_time_eq;
-use zeroize::Zeroize;
+
+/* max algo right now is SHA3_224 with 144 byts blocksize,
+ * use slightly larger for good measure (and alignment) */
+const IPAD_INIT: [u8; 160] = [0x36; 160];
+const OPAD_INIT: [u8; 160] = [0x5c; 160];
```

Comment:
should we make the number 160 constant instead of using it on at least 5 different places here and below?